eox_tenant/apps.py

```diff
@@ -23,12 +23,14 @@ class EdunextOpenedxExtensionsTenantConfig(AppConfig):
             'lms.djangoapp': {
                 'test': {'relative_path': 'settings.test'},
                 'common': {'relative_path': 'settings.common'},
-                'aws': {'relative_path': 'settings.aws'},
+                'aws': {'relative_path': 'settings.production'},
+                'production': {'relative_path': 'settings.production'},
             },
             'cms.djangoapp': {
                 'test': {'relative_path': 'settings.test'},
                 'common': {'relative_path': 'settings.common'},
-                'aws': {'relative_path': 'settings.aws'},
+                'aws': {'relative_path': 'settings.production'},
```

Comment:
I think that this is counterintuitive, maybe it would be better to have an aws.py file that imports all from production.py

Reply:
@morenol @felipemontoya  what do you think if we remove the aws file, I mean no more hawthorn support ?

Reply:
Yes, I think that we can do that, in any case we should add an entry in the changelog

Reply:
@morenol so, won't we support to ironwood?

Reply:
No, Irowood also uses the production.py setting